### PR TITLE
Show Profile User(simple and supervisor)

### PR DESCRIPTION
**Pull request by samnangcattor**

### CODE REVIEW

File: app/controllers/supervisor/users_controller.rb [hunk 1]
```diff
@@ -14,10 +17,23 @@ def create
...
+    @user = User.find params[:id]
   end
 
   private
   def user_params
     params.require(:user).permit :name, :email, :password, :password_confirmation
   end
+
+  def logged_in_user
+      unless logged_in?
```
Comment (HorNorin) [inline app/controllers/supervisor/users_controller.rb]: 4 space indent

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -36,4 +40,13 @@ def log_out
     session.delete :user_id
     @current_user = nil
   end
+
+  def redirect_back_or default
+      redirect_to(session[:forwarding_url] || default)
```
Comment (HorNorin) [inline app/helpers/sessions_helper.rb]: 4 space indent

File: app/helpers/sessions_helper.rb [hunk 3]
```diff
@@ -36,4 +40,13 @@ def log_out
...
+      session.delete :forwarding_url
+  end
+
+  def store_location
+      session[:forwarding_url] = request.url if request.get?
```
Comment (HorNorin) [inline app/helpers/sessions_helper.rb]: 4 space indent use 2 instead

**Comment by phornsopheak** [comment]
it's ok

File: config/locales/database.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# MySQL.  Versions 5.0+ are recommended.
```
Comment (dieunb) [inline config/locales/database.yml]: remove this file, don't push it

File: app/helpers/sessions_helper.rb [hunk 5]
```diff
@@ -36,4 +40,13 @@ def log_out
     session.delete :user_id
     @current_user = nil
   end
+
+  def redirect_back_or default
+    redirect_to(session[:forwarding_url] || default)
```
Comment (dieunb) [inline app/helpers/sessions_helper.rb]: remove `()`

**Pull request merged**